_data/compose-cli/docker_compose.yaml

```
@@ -265,7 +265,7 @@ options:
   description: |-
     DEPRECATED! USE --project-directory INSTEAD.
     Specify an alternate working directory
-    (default: the path of the Compose file)
+    (default: the path of the, first specified, Compose file)
```

```suggestion
    (default: the path of the first Compose file specified). See also, [Understanding multiple compose files](../../../compose/extends/#understanding-multiple-compose-files).
```
@glours could you check pls if the relative path works.